modules/get-started/pages/rpk-first-steps.adoc

```diff
@@ -0,0 +1,240 @@
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-describe.adoc[].
+
+== Consume records from a topic
+
+Consume records from a topic to verify the data produced.
+
+[source,bash]
+----
+rpk topic consume tutorial
+----
+
+Output:
+[source,json]
+----
+{ "topic": "tutorial", "value": "hello", "timestamp": 1678807229837, "partition": 0, "offset": 0 }
+{ "topic": "tutorial", "value": "world", "timestamp": 1678807232413, "partition": 0, "offset": 1 }
+----
+
+Consuming from an offset, where `2` is not inclusive: 
+[source,bash]
+----
+rpk topic consume tutorial --offset 0:2
+----
+Output:
+[source,json]
+----
+{ "topic": "tutorial", "value": "hello", "timestamp": 1678807229837, "partition": 0, "offset": 0 }
+{ "topic": "tutorial", "value": "world", "timestamp": 1678807232413, "partition": 0, "offset": 1 }
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-consume.adoc[].
+
+== Create consumer groups and consuming topics
+
+Organize consumers into groups to share workloads and balance consumption.
+
+[source,bash]
+----
+rpk topic consume tutorial --group tutorial-group
+----
+
+NOTE: The consumer group is created when you consume from a topic.
+
+Output:
+[source,json]
+----
+{
+  "topic": "tutorial",
+  "value": "hello",
+  "timestamp": 1734640650348,
+  "partition": 0,
+  "offset": 0
+}
+{
+  "topic": "tutorial",
+  "value": "world",
+  "timestamp": 1734640653558,
+  "partition": 0,
+  "offset": 1
+}
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-consume.adoc[].
+
+== List all consumer groups
+
+List available consumer groups in your cluster.
+
+[source,bash]
+----
+rpk group list
+----
+
+Output:
+[source,bash]
+----
+BROKER  GROUP           STATE
+0       tutorial-group  Empty
+----
+
+See xref:reference:rpk/rpk-group/rpk-group-list.adoc[].
+
+== Get a description of a consumer group
+
+View details about the consumer group’s state, coordinator, members, and offsets.
+
+[source,bash]
+----
+rpk group describe tutorial-group
+----
+Output:
+[source,bash]
+----
+GROUP        tutorial-group
+COORDINATOR  0
+STATE        Empty
+BALANCER
+MEMBERS      0
+TOTAL-LAG    0
+
+TOPIC     PARTITION  CURRENT-OFFSET  LOG-START-OFFSET  LOG-END-OFFSET  LAG   MEMBER-ID  CLIENT-ID  HOST
+tutorial  0          2               0                 2               0
+----
+
+See xref:reference:rpk/rpk-group/rpk-group-describe.adoc[].
+
+== Delete a consumer group
+
+Clean up by removing the tutorial consumer group once you’re done.
+
+[source,bash]
+----
+rpk group delete tutorial-group
+----
+Output:
+[source,bash]
+----
+GROUP           STATUS
+tutorial-group  OK
+----
+
+See xref:reference:rpk/rpk-group/rpk-group-delete.adoc[].
+
+== Delete a topic
+
+Clean up by removing the tutorial topic once you’re done.
+
+[source,bash]
+----
+rpk topic delete tutorial
+----
+Output:
+[source,bash]
+----
+TOPIC     STATUS
+tutorial  OK
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-delete.adoc[].
+
+== Next steps
```

Comment:
If you don't merge this with rpk Profile, then add a link to it here. 
Also, you should add a link to this page in the Intro to rpk page and the Install/Update rpk page!